<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="lookup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>831</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>시군</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>사업장명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>소재지</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>면적(제곱미터)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>용량(세제곱미터)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>침출수처리</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>사용</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>비고</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="push_b">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>검색하기</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>전체</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>시군</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사업장명</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>소재지</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>면적</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>용량</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>침출수처리</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사용</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>비고</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="search_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="total_b">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>전체보기</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>831</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph_verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="add_b">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>추가하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_b">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제하기</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
